models/action.go

```diff
@@ -136,6 +137,16 @@ func (a Action) GetIssueInfos() []string {
 	return strings.SplitN(a.Content, "|", 2)
 }
 
+func (a Action) GetIssueTitle() string {
+	issueID := com.StrTo(a.GetIssueInfos()[0]).MustInt64()
```

Comment:
As I understood index ```0``` is the issue id. At the next line I call ```GetIssueById```.
What's wrong?